src/tests/unit/services/PostEventProcessor.test.ts

```diff
@@ -337,26 +385,59 @@ describe("PostEventProcessor", () => {
 			// eslint-disable-next-line @typescript-eslint/unbound-method
-			expect(mockIprService.saveEventData).toHaveBeenCalledWith("01333e01-dde3-412f-a484-4444", "SET journeyWentAsyncOn = :journeyWentAsyncOn, expiresOn = :expiresOn, documentType = :documentType, clientSessionId = :clientSessionId", { ":journeyWentAsyncOn": 1681902001, ":expiresOn": expiresOn, ":documentType": "PASSPORT", ":clientSessionId": "asdfadsfasdf" });
+			expect(mockIprServiceSession.saveEventData).toHaveBeenCalledWith("01333e01-dde3-412f-a484-4444", "SET journeyWentAsyncOn = :journeyWentAsyncOn, expiresOn = :expiresOn, ipvStartedOn = :ipvStartedOn, userEmail = :userEmail, clientName = :clientName, redirectUri = :redirectUri, documentType = :documentType, clientSessionId = :clientSessionId", { 
+				":journeyWentAsyncOn": 1681902001, 
+				":clientName": "test",
+				":ipvStartedOn": "test",
+				":redirectUri": "test",
+				":userEmail": "test@digital.cabinet-office.gov.uk",
+				":expiresOn": expiresOn,
+				":documentType": "PASSPORT",
+				":clientSessionId": "asdfadsfasdf",
+			});
 			// eslint-disable-next-line @typescript-eslint/unbound-method
 			expect(mockLogger.info).toHaveBeenNthCalledWith(3, "No post_office_details in F2F_YOTI_START event");
 		});
 
 		it("Logs if post_office_details and document_details is missing", async () => {
+			const Item = {
+				userId: "01333e01-dde3-412f-a484-4444",
+				userEmail: "test@digital.cabinet-office.gov.uk",
+				ipvStartedOn: "test",
+				clientName: "test",
+				redirectUri: "test",
+				expiresOn: absoluteTimeNow() + 1000,
+			};
+			mockDynamoDbClient.send = jest.fn().mockResolvedValue({ Item });
 			const expiresOn = absoluteTimeNow() + Number(process.env.SESSION_RETURN_RECORD_TTL_SECS!);
-			await postEventProcessor.processRequest(VALID_F2F_YOTI_START_TXMA_EVENT_STRING);
+			await postEventProcessorMockSessionService.processRequest(VALID_F2F_YOTI_START_TXMA_EVENT_STRING);
 			// eslint-disable-next-line @typescript-eslint/unbound-method
-			expect(mockIprService.saveEventData).toHaveBeenCalledWith("01333e01-dde3-412f-a484-4444", "SET journeyWentAsyncOn = :journeyWentAsyncOn, expiresOn = :expiresOn", { ":journeyWentAsyncOn": 1681902001, ":expiresOn": expiresOn });
+			expect(mockIprServiceSession.saveEventData).toHaveBeenCalledWith("01333e01-dde3-412f-a484-4444", "SET journeyWentAsyncOn = :journeyWentAsyncOn, expiresOn = :expiresOn, ipvStartedOn = :ipvStartedOn, userEmail = :userEmail, clientName = :clientName, redirectUri = :redirectUri", { 
+				":journeyWentAsyncOn": 1681902001, 
+				":clientName": "test",
+				":ipvStartedOn": "test",
+				":redirectUri": "test",
+				":userEmail": "test@digital.cabinet-office.gov.uk",
+				":expiresOn": expiresOn,
+			});
 			// eslint-disable-next-line @typescript-eslint/unbound-method
 			expect(mockLogger.info).toHaveBeenNthCalledWith(3, "No post_office_details in F2F_YOTI_START event");
```

Comment:
We need a unit-test to check if AUTH event wasnt received and no record was found in AuthTable, it should end up logging the message and throwing the error as expected.